src/EnergyPlus/SingleDuct.hh

```diff
@@ -323,6 +323,7 @@ namespace SingleDuct {
         Real64 DesignPrimaryAirVolRate;                           // System sizing adjustments, filled from design OA spec using sizing mode flags.
         DataZoneEquipment::PerPersonVentRateMode OAPerPersonMode; // mode for how per person rates are determined, DCV or design.
         bool printWarning;                                        // flag to print warnings only once
+        Real64 ADEff = 1.0;
```

Comment:
I am questioning where this saved data should live. The DesignSpecification:ZoneAirDistribution object name can be entered in the Sizing:Zone object. This object name can also be entered in Controller:MechanicalVentilation. So somewhere this data needs to be saved. Also, there is an effectiveness for both cooling and heating, so using the min of these to determine OA seems wrong. If cooling eff=1 and heating eff=0.8 then OA during heating and cooling should be different.